### PR TITLE
Include Zone Name in ETL

### PR DESCRIPTION
**Pull request by Charlie-Henry**
https://github.com/cityofaustin/atd-data-tech/issues/11431

Jeremy provided me a list of ID ranges (for kisoks and app zones) for each parking location group. These are used by finance to report totals out for each location. We were using a spreadsheet lookup table that was getting loaded into Power BI to do this previously. This should require less maintenance on our part going forward as new meters are added or moved around locations they *should* be within the ranges provided by Jeremy. 

I added a location_name column to the passport data, flowbird data, and the flowbird payment data tables. This has been also added to two datasets in Socrata, the main parking transactions dataset and one of the ones used by Finance. 

I also refactored the socrata publishing code.

A big benefit of this work is that we can group parking kiosk and app parking data into the same categories, which required a ton of work to do previously.

We'll have to re-process all of the data to supply these groupings.

### CODE REVIEW

File: meters/smartfolio_s3.py [hunk 1]
```diff
@@ -274,10 +285,11 @@ def to_postgres(smartfolio):
             "start_time",
             "end_time",
             "amount",
+            "location_name",
         ]
     ]
 
-    smartfolio.loc[:, "source"] = "Parking Meters"
+    smartfolio["source"] = "Parking Meters"
```
Comment (Charlie-Henry) [inline meters/smartfolio_s3.py:292]: idk why I had this written this way previously 🤦‍♂️

File: meters/smartfolio_s3.py [hunk 2]
```diff
@@ -157,6 +158,12 @@ def postgres_datetime(time_field):
     )
     return str(output)
 
+def create_location_name(row):
+    id = row["meter_id"]
+    for id_range in METER_LOCATION_NAMES:
+        if id in id_range:
+            return METER_LOCATION_NAMES[id_range]
+    return "Unknown Location"
```
Comment (Charlie-Henry) [inline meters/smartfolio_s3.py:166]: `Unknown Location` will help us debug issues in the future hopefully.

**Review by chiaberry — APPROVED** [review]
🚢

**Review by johnclary — APPROVED** [review]
nice cleanup! glad to see a better source for these locations. thanks @Charlie-Henry! 

🚢 🚢 🚢 🚢

**Review by roseeichelmann — APPROVED** [review]
looks good! ✨

**Pull request merged**